compiler/src/iree/compiler/API/Internal/IREEGPUDialectCAPI.cpp

```diff
@@ -1,120 +1,120 @@
-}
-
-MlirAttribute ireeGPUPipelineOptionsAttrGetNoReduceSharedMemoryBankConflicts(
-    MlirAttribute attr) {
-  auto gpuAttr =
-      llvm::cast<mlir::iree_compiler::IREE::GPU::GPUPipelineOptionsAttr>(
-          unwrap(attr));
-  return wrap(gpuAttr.getNoReduceSharedMemoryBankConflicts());
-}
-
-MlirAttribute
-ireeGPUPipelineOptionsAttrGetReorderWorkgroupsStrategy(MlirAttribute attr) {
-  auto gpuAttr =
-      llvm::cast<mlir::iree_compiler::IREE::GPU::GPUPipelineOptionsAttr>(
-          unwrap(attr));
-  return wrap(gpuAttr.getReorderWorkgroupsStrategy());
-}
-
-MlirTypeID ireeGPUPipelineOptionsAttrGetTypeID() {
-  return wrap(
-      mlir::iree_compiler::IREE::GPU::GPUPipelineOptionsAttr::getTypeID());
-}
-
-static_assert(
-    static_cast<uint32_t>(ireeGPUReorderWorkgroupsStrategyEnumNone) ==
-            static_cast<uint32_t>(mlir::iree_compiler::IREE::GPU::
-                                      ReorderWorkgroupsStrategy::None) &&
-        static_cast<uint32_t>(ireeGPUReorderWorkgroupsStrategyEnumSwizzle) ==
-            static_cast<uint32_t>(mlir::iree_compiler::IREE::GPU::
-                                      ReorderWorkgroupsStrategy::Swizzle) &&
-        static_cast<uint32_t>(ireeGPUReorderWorkgroupsStrategyEnumTranspose) ==
-            static_cast<uint32_t>(mlir::iree_compiler::IREE::GPU::
-                                      ReorderWorkgroupsStrategy::Transpose) &&
-        static_cast<uint32_t>(ireeGPUReorderWorkgroupsStrategyEnumTranspose) ==
-            mlir::iree_compiler::IREE::GPU::
-                getMaxEnumValForReorderWorkgroupsStrategy(),
-    "ireeGPUReorderWorkgroupsStrategyEnum and "
-    "mlir::iree_compiler::IREE::GPU::ReorderWorkgroupsStrategy definitions "
-    "have diverged");
-
-bool ireeAttributeIsAGPUReorderWorkgroupsStrategyAttr(MlirAttribute attr) {
-  return llvm::isa<
-      mlir::iree_compiler::IREE::GPU::ReorderWorkgroupsStrategyAttr>(
-      unwrap(attr));
-}
-
-MlirTypeID ireeGPUReorderWorkgroupsStrategyAttrGetTypeID() {
-  return wrap(mlir::iree_compiler::IREE::GPU::ReorderWorkgroupsStrategyAttr::
-                  getTypeID());
-}
-
-MlirAttribute ireeGPUReorderWorkgroupsStrategyAttrGet(
-    MlirContext mlirCtx, ireeGPUReorderWorkgroupsStrategyEnum value) {
-  mlir::MLIRContext *ctx = unwrap(mlirCtx);
-  return wrap(
-      mlir::iree_compiler::IREE::GPU::ReorderWorkgroupsStrategyAttr::get(
-          ctx, static_cast<
-                   mlir::iree_compiler::IREE::GPU::ReorderWorkgroupsStrategy>(
-                   value)));
-}
-
-ireeGPUReorderWorkgroupsStrategyEnum
-ireeGPUReorderWorkgroupsStrategyAttrGetValue(MlirAttribute attr) {
-  assert(ireeAttributeIsAGPUReorderWorkgroupsStrategyAttr(attr) &&
-         "attr is not a GPUReorderWorkgroupsStrategyAttr");
-  return static_cast<ireeGPUReorderWorkgroupsStrategyEnum>(
-      llvm::cast<mlir::iree_compiler::IREE::GPU::ReorderWorkgroupsStrategyAttr>(
-          unwrap(attr))
-          .getValue());
-}
+// Copyright 2024 The IREE Authors
```

Comment:
What is this diff, I don't see anything changed?

Reply:
someone checked in bad line endings
![image](https://github.com/user-attachments/assets/5f282642-a50f-4a95-b3a9-91c80c04b98e)

Reply:
Wow github does not render that for me, it just looks like -120 +120 of the same stuff